packages/fiori/src/themes/NavigationLayout.css

```diff
@@ -0,0 +1,90 @@
+
+:host(:not([hidden])) {
+	width: 100%;
+	height: 100%;
+	display: block;
+	padding: 0.5rem 0.5rem 0 0.5rem;
+	background: var(--sapBackgroundColor);
+	box-sizing: border-box;
+	overflow: hidden;
+}
+
+:host(:not([hidden][is-tablet])) {
+	padding: 0.5rem;
+}
+
+:host(:not([hidden][is-phone])) {
+	padding: 0.25rem;
+}
+
+.ui5-nl-root {
+	width: 100%;
+	height: 100%;
+	display: flex;
+	flex-direction: column;
+	box-sizing: border-box;
+}
+
+.ui5-nl-header {
+
+}
+
+.ui5-nl-section {
+	display: flex;
+	flex: 1;
+	min-height: 0;
+	position: relative;
+}
+
+.ui5-nl-aside {
+	margin: 0.5rem 0.5rem 0 0;
+	transition: transform 0.3s;
+	z-index: 1;
+}
+
+:host([is-phone]) .ui5-nl-aside,
+:host([side-collapsed]) .ui5-nl-aside {
+	z-index: 3;
+}
+
+:host([is-phone]) .ui5-nl-aside {
+	margin: 0.25rem 0.25rem 0 0;
+}
+
+:host([is-phone]) .ui5-nl-aside {
+	position: absolute;
+	top: 0;
+	left: 0;
+	bottom: 0;
+	width: 100%;
+}
+
+.ui5-nl-content {
+	flex: 1;
+	min-width: 0;
+	margin: 0.5rem 0 0 0;
+	z-index: 2;
+
+	box-shadow: var(--_ui5_nl_box_shadow);
+	border-radius: 0.5rem 0.5rem 0 0;
+	background: var(--sapBackgroundColor);
+}
+
+:host([is-phone]) .ui5-nl-content {
+	margin: 0.25rem 0 0 0;
+}
+
+:host([is-phone]) .ui5-nl-content,
+:host([is-tablet]) .ui5-nl-content {
+	border-radius: 0.5rem;
+}
+
+:host([side-collapsed][is-phone]) .ui5-nl-aside {
+	transform: translateX(calc(-100% - 0.25rem));
+}
+
+::slotted([ui5-toolbar]),
```

Comment:
- if you put toolbar in the content - it also automatically becomes rounded, needs a more specific css selector
- you can not easily put a tab-container under the shellbar like this (which is part of the design). Would be possible if app developer overrides the shellbar styles, which could be fine...